source/administration/product-receipe-template.rst

```diff
@@ -0,0 +1,62 @@
+Product Recipe template
+========================
+
+This template defined how a product recipe can be added for the depending
+people.
+
+Problem Statement
+-------------------
+
+Define the problem statement. The problem statement could be anything from adding
+an architecture setup or installations.
+
+The problem statement should be written from user perspective and from organisation
+perspective.
+
+The user perspective might include
+
+- The requirements of the user
+- The environment of the user
+- The dependecies the user is having
+- The user case user would run the installation in
+
+The organisation perspective might include
+
+- How much effort the organisations adds in implementing the solution
+- How many work hours are put on implementing the solution for the user
+- Listing team putting the solution for the user
+- Dependecies with ongoing installations
+
+Pre-installation/pre-configuration notes
+----------------------------------------
+
+Define the pre-installation information. Projects installation has pre-installation
+steps describing the things you should setup when installating a requirement.
+Breifly describe the pre-installation or pre-configuration notes. Things you
+might include
+
+- Environment information
+- Version information
+- Release information
+- Dependecy information
+- Rollback information
+- Line of Business information
+
+Instructions Setup and Configuring the Solution
+-----------------------------------------------
+
+Define the instructions setup instructions. The instructions should be detailed
+and have everything about setting up and configuring the solution. Things you
+might include
+
+- Installation instructions
+- Trouble shooting instructions
+- Testing the installation
+- Testing the installation in multiple environments
+
+Defining Recipe Category For Search and Retrival
+------------------------------------------------
+
+Define the categories and tag for the users. The users should be easily search
+and retrieve the receipe. Add tech, line of business, architecture, environement
+information.
```

Comment:
Please help make this paragraph a little clearer.

Reply:
If you can tell me what tags you have in your organisation for this sort of implementation, I can include the details. I added few generic things in these lines that could let any member of mattermost org implement a product recipe and add proper tags in the documentation.